src/OrchardCore.Modules/OrchardCore.Forms/placement.json

```diff
@@ -0,0 +1,11 @@
+{
+    "TitlePart": [
```

Comment:
Why is is necessary since this can be done with the settings now?

Reply:
To always hide? - regardless of setting. May be we can remove placement  entry for Form but not for Label